unity/CAVE/Assets/W3D_Translator/Classes.cs

```diff
@@ -131,14 +125,13 @@ public string colorString
         public string soundRef;
 
         [XmlElement(ElementName="Placement")]
-        public PlacementType Placement;
+        public Placement Placement;
```

Comment:
Do you run into any naming collisions for using the same name for the type and the variable?

Reply:
I noticed in some cases the variable name is in camelcase which would make this something like `public Placement placement`. 

I saw later in the code there was `public ModeType modeType;`

Reply:
The only name collisions I'm running into is when Unity has a class with the same name but the variable name isn't an issue as well. And even that has actually been good because it points me in the right direction to the kind of Unity objects I should be translating the class to

Reply:
I had been doing the naming based on whether the property was a generated class or a base type but it looks like naming conventions call for PascalCase always so I will refactor

Reply:
Ended up doing a whole bunch of renaming. I left the `[xmlText]` variables as lowercase because I'm not really going to need to use them, made the enum declarations plural, and a couple other odds and ends